packages/repack/src/commands/start.ts

```diff
@@ -192,9 +193,17 @@ export async function start(_: string[], config: Config, args: StartArguments) {
 
   await start();
 
+  let sendEventsStop = () => {};
+  if (sendEventsArg) {
```

Comment:
In the future, we should consider sending this event on a 5 second interval rather than a single time only on startup. That would allow us to reconnect to a sleeper app that is closed and re-opened, or to start this process first and then sleeper afterward.

Reply:
Actually - following up on our conversation - this should be done in sleeper-mini-core in the sendEvents code.